specs/signature-specification.md

```diff
@@ -43,6 +43,7 @@ Signature Manifest Example
 - **`blobs`** (*array of objects*): This REQUIRED property contains collection of only one [artifact descriptor][artifact-descriptor] referencing signature envelope.
   - **`mediaType`** (*string*): This REQUIRED property contains media type of signature envelope blob. Following values are supported
     - `application/jose+json`
```

Comment:
This seems like maybe not correct? are you saying that compact representation is NOT allowed? thats how i read them.